src/data/faq.json

```diff
@@ -101,6 +101,170 @@
+                            "</ul>"
+                        ]
+                    },
+                    {
+                        "title": "What is the digital certificate of vaccination?",
+                        "anchor": "vaccination_certificate_what",
+                        "active": true,
+                        "textblock": [
+                            "Starting with version 2.3 of the Corona-Warn-App (CWA), you can add your digital vaccination certificate in the CWA and thus securely and digitally prove your Corona vaccination status via 2D barcode (QR code). The CWA indicates the complete vaccination protection 14 days after the last required vaccination. The digital vaccination certificate is an additional way to document vaccinations. Thereby you are able to digitally store information such as vaccination time and vaccine in a personalised manner on their smartphones. The use of digital certificate of vaccination in the CWA is voluntary. You can also prove your Corona vaccination in a different way."
+                        ]
+                    },
+                    {
+                        "title": "",
+                        "anchor": "vaccination_certificate_why",
+                        "active": true,
+                        "textblock": [
+                            ""
+                        ]
+                    },
+                    {
+                        "title": "",
+                        "anchor": "vaccination_certificate_who",
+                        "active": true,
+                        "textblock": [
+                            ""
+                        ]
+                    },
+                    {
+                        "title": "",
+                        "anchor": "vaccination_certificate_privileges_travel",
+                        "active": true,
+                        "textblock": [
+                            ""
+                        ]
+                    },
+                    {
+                        "title": "",
+                        "anchor": "vaccination_certificate_how",
+                        "active": true,
+                        "textblock": [
+                            ""
+                        ]
+                    },

```

Comment:
These entries are all missing content and title text. This results in blank boxes being shown on the website.

The PR should be put into draft status until this is fixed, since it is not ready for publication.

Reply:
added EN text now

Reply:
article **Can I use a complete vaccination status in the app as a travel permission?** is still tentative...